editors/vscode/src/common.ts

```diff
@@ -259,3 +292,36 @@ async function maybeSendStartupTelemetryEvent(
 
     telemetryLogger.logUsage("extension-activated", usageData);
 }
+
+const HELP_URL = "https://docs.slint.dev/latest/docs/slint/reference/";
+
+function getHelpUrlForElement(elementName: string): string | null {
```

Comment:
I'm slightly concerned about "bitrot" there and things getting out of sync or missing.